docs/admin-manual/install/install-using-docker.md

```diff
@@ -14,6 +14,12 @@ This guide walk you though the steps required to install LibreTime on your syste
 
 Before installing LibreTime, you need to make sure that [Docker](https://docs.docker.com/engine/) is installed on your operating system and **up-to-date**.
 
+For a quick install of Docker, you can execute the command below:
+
+```bash
+curl -sSL https://get.docker.com/ | CHANNEL=stable bash
```

Comment:
This is not a good way of installing software. Piping random things from the Internet to bash is never a good solution